filtercascade/__init__.py

```diff
@@ -113,22 +156,49 @@ def from_buf(cls, buf):
 
 
 class FilterCascade:
-    DIFF_FMT = b"<III"
-    VERSION_FMT = b"<H"
+    # The metadata struct is three values defining the configuration settings
+    # used in the production of the filter cascade, per-level.
+    # bytes 0-3: filtersize in bits as an unsigned int
```

Comment:
Maybe we should say "little endian" so I don't have to look up what "<" means each time :D